packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/dedicatedCloud-datacenter.html

```diff
@@ -1,118 +1,118 @@
-<div class="dedicatedCloud-datacenter">
-    <div
-        class="text-center mt-5"
-        data-ng-if="$ctrl.loadingInformations && !$ctrl.loadingError"
-    >
-        <oui-spinner data-size="l"></oui-spinner>
-    </div>
+<div
```

Comment:
Please don't look at this whole file, it's mostly whitespaces, you can suppress them by clicking on "Hide whitespace changes" 🙏